packages/admin-ui/src/Card/Card.tsx

```diff
@@ -1,56 +1,104 @@
+const CardHeaderBase = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(
     ({ className, ...props }, ref) => (
-        <h3
-            ref={ref}
-            className={cn("text-2xl font-semibold leading-none tracking-tight", className)}
-            {...props}
-        />
+        <div ref={ref} className={cn("flex flex-col space-y-1.5", className)} {...props} />
     )
 );
-CardTitle.displayName = "CardTitle";
+CardHeaderBase.displayName = "CardHeader";
+
+const CardHeader = makeDecoratable("CardHeader", CardHeaderBase);
 
-const CardDescription = React.forwardRef<
+const CardTitleBase = React.forwardRef<
+    HTMLParagraphElement,
+    React.HTMLAttributes<HTMLHeadingElement>
+>(({ className, ...props }, ref) => (
+    <h3
+        ref={ref}
+        className={cn("text-2xl font-semibold leading-none tracking-tight", className)}
+        {...props}
+    />
+));
+CardTitleBase.displayName = "CardTitle";
+
+const CardTitle = makeDecoratable("CardTitle", CardTitleBase);
+
+const CardDescriptionBase = React.forwardRef<
     HTMLParagraphElement,
     React.HTMLAttributes<HTMLParagraphElement>
 >(({ className, ...props }, ref) => (
     <p ref={ref} className={cn("text-sm text-muted-foreground", className)} {...props} />
 ));
-CardDescription.displayName = "CardDescription";
+CardDescriptionBase.displayName = "CardDescription";
 
-const CardContent = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(
-    ({ className, ...props }, ref) => (
-        <div ref={ref} className={cn("p-6 pt-0", className)} {...props} />
-    )
+const CardDescription = makeDecoratable("CardDescription", CardDescriptionBase);
+
+const CardContentBase = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(
+    ({ className, ...props }, ref) => <div ref={ref} className={cn("", className)} {...props} />
 );
-CardContent.displayName = "CardContent";
+CardContentBase.displayName = "CardContent";
+
+const CardContent = makeDecoratable("CardContent", CardContentBase);
 
-const CardFooter = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(
+const CardFooterBase = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(
     ({ className, ...props }, ref) => (
-        <div ref={ref} className={cn("flex items-center p-6 pt-0", className)} {...props} />
+        <div ref={ref} className={cn("flex items-center", className)} {...props} />
     )
 );
-CardFooter.displayName = "CardFooter";
+CardFooterBase.displayName = "CardFooter";
+
+const CardFooter = makeDecoratable("CardFooter", CardFooterBase);
+
+interface CardProps extends Omit<CardRootProps, "content"> {
+    headerTitle?: React.ReactElement<typeof CardTitleBase>;
+    headerDescription?: React.ReactElement<typeof CardDescriptionBase>;
+    content?: React.ReactElement<typeof CardContentBase>;
+    footer?: React.ReactElement<typeof CardFooterBase>;
+}
```

Comment:
I don't see any references to some variants defined in the Figma file, such as: 

- Elevation
- Border Radius
- Paddings

Is it intentional?

Reply:
Added.